projects.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: Projects
+parent: Get Involved
+order: 1
+---
+
+Please ask in #welcome:tomesh.net if you would like to learn more or contribute to any one of these projects.
+
+## Nodes/Network
+
+### Raspberry/Orange Pi Install Script + Bi-Weekly Virtual Mesh
+Description: Installation script for Raspbery and Orange Pi devices from a stock firmware.   
+Status: See [Github Issues](https://github.com/tomeshnet/prototype-cjdns-pi/issues)  
+Contact: benhylau  
+[More Info](https://github.com/tomeshnet/prototype-cjdns-pi2)
+
+
+### Mesh Orange
+Description: Ramdisk based os for keeping a lean and mean node on Orange Pis  
+Status: Work in progess https://github.com/tomeshnet/mesh-orange/issues/1  
+Contact: benhylau  
+[More Info](https://github.com/tomeshnet/mesh-orange)  
+
+### Mesh ISP
+Status: Ethermesh did some demo of this at the conference and one of the mesh locals in the US is doing something similar  
+Contact: benhylau (until someone takes over)  
+[More Info](https://github.com/tomeshnet/mesh-isp)  
+
+### Outernet / RachelPi OS
+Status: No one is actively working on this but benhylau tried to set up Outernet with charlie from nycmesh, and spoke with someone from sudomesh about hosting wikipedia and khan with RACHEL  
+Contact: benhylau (until someone takes over)  
+More Info: [OuterNet](https://outernet.is)  
+More Info: [RachelOffline](https://racheloffline.org)  
+
+## Resources
```

Comment:
Unclear how this section is different than the "Get Involved" parent page?

Reply:
I think get involved needs some TLC as well and make it allot easier to follow. The detailed stuff (like how to use WEKA) may be worth having some where else. Dont want to scare people off when they first come here:)